durango/src/error.rs

```diff
@@ -0,0 +1,102 @@
+//! The Durango error
```

Comment:
Shouldn't this be `veraucruz-client`?

Reply:
I thought this was deleted?

Reply:
Yeah me too :crying_cat_face:
Some references to Sinaloa were also introduced by the rebase